template.go

```diff
@@ -310,20 +320,30 @@ func _getTemplate(t0 *template.Template, root string, subMods [][]string, others
 			for _, otherFile := range others {
 				var data []byte
 				fileAbsPath := filepath.Join(root, otherFile)
-				data, err = ioutil.ReadFile(fileAbsPath)
+				f, err := fs.Open(fileAbsPath)
+				if err != nil {
+					f.Close()
+					continue
+				}
+				data, err = ioutil.ReadAll(f)
+				f.Close()
 				if err != nil {
+					logs.Trace("template parse file err:", err)
```

Comment:
Can you please use more descriptif error messages:
```suggestion
					logs.Trace("template file parse error:", err)
```